pkg/cloud/rgraph/rnode/action_update_test.go

```diff
@@ -59,12 +59,13 @@ func createNotExistingFakeNode() Node {
 	return &fn
 }
 
-func dropRefEventList(toRefs []string) exec.EventList {
+func updateEventList(toRefs []string) exec.EventList {
```

Comment:
nit: you may consider using variadic arguments. It may require changes to other helpers for consistency

Reply:
yes, great idea, fixed